### ISSUE TITLE
Шаги для установки фильтров на список не работают для перечислений и для дат

### ISSUE THREAD

**Issue by akademik1986** [Pr-Mex/vanessa-automation#1741]
Версия Vanessa Automation 1.2.039.4
Версия платформы 8.3.20
Режим совместимости базы, где запускается Vanessa Automation 8.3.17
Операционная система windows

1. Если использовать шаги установки фильтров для перечислений, такие как

		И для каждой строки таблицы отбора я устанавливаю отбор в списке
			| 'Состояние'    | 'Равно' | 'Выполнен' |
		И Я устанавливаю фильтр на список
			| 'Состояние'    | 'Равно' | 'Выполнен' |
			
то при выполнении шага будет ошибка, что значение не может быть выбрано. 
![image](https://user-images.githubusercontent.com/19662685/185618642-30b7a4e2-0d55-4a69-8a94-fe24df0fc644.png)
Выбрать перечисление можно только так
		И я нажимаю кнопку выбора у поля с именем "Имя поля"
		И я выбираю из списка 'Выполнено'

2. С этими шагами по установке фильтров связана еще одна ошибка. что нельзя установить ими отбор по дате, так как при установке отбора к полю с типом дата платформа предлагает сначала выбрать тип периода, а потом только указать дату
![image](https://user-images.githubusercontent.com/19662685/185620327-e79069b9-ad5c-498c-8fe6-f8c374326400.png)

		И для каждой строки таблицы отбора я устанавливаю отбор в списке
			| 'Дата'    | 'Равно' | '01.01.2022' |			
		И Я устанавливаю фильтр на список
			| 'Дата'    | 'Равно' | '01.01.2022' |			

![image](https://user-images.githubusercontent.com/19662685/185620500-f36dd7b6-7774-41e1-9584-cba6eabd30c5.png)

**Comment by Pr-Mex** [comment]
@akademik1986 
Да, думаю можно эти шаги научить работать с перечислениями и датами.
